lib/actions/push.ts

```diff
@@ -44,10 +44,9 @@ function getBuildTarget(appOrDevice: string): BuildTarget | null {
 }
 
 async function getAppOwner(sdk: BalenaSDK, appName: string) {
-	const {
-		exitWithExpectedError,
-		selectFromList,
-	} = await import('../utils/patterns');
+	const { exitWithExpectedError, selectFromList } = await import(
+		'../utils/patterns'
+	);
```

Comment:
This is [prettier](https://www.npmjs.com/package/prettier)'s doing. Maybe a new version of prettier got released, or something. It was actually required in order for the Travis build to pass (as the build checks whether all source files "conform" with what prettier would do if it was run).